elements/crash-modal.js

```diff
@@ -0,0 +1,83 @@
+const Modal = require('base-elements/modal')
+const remote = require('electron').remote
+const html = require('choo/html')
+const css = require('sheetify')
+
+const button = require('./button')
+
+const prefix = css`
+  :host {
+    min-width: 20rem;
+    max-width: 25rem;
+    background-color: var(--color-white);
+    box-shadow: 0 1.2rem 2.4rem rgba(0,0,0,.5);
+  }
+  :host .exit {
+    border: none;
+    color: var(--color-neutral-40);
+  }
+  :host .exit:hover,
+  :host .exit:focus {
+    color: var(--color-neutral);
+  }
+  :host .icon-cross {
+    vertical-align: middle;
+    width: 1.1em;
+    max-height: 1.6em;
+    transition: color .025s ease-out;
+  }
+`
+
+module.exports = createWidget
+
+function createWidget () {
+  const modal = Modal({ render, onexit })
+
+  return modal
+
+  function onexit () {
+    var window = remote.getCurrentWindow()
+    window.close()
+  }
+
+  function render (onOk, onExit) {
+    return html`
+      <div class="relative flex flex-column justify-center ${prefix}">
+        <section class="pa4">
+          <h3 class="f4">Unexpected issue</h3>
+          <p class="mt3 mb4 f7 color-neutral-70">
+            There was an unexpected issue. We will need to close the app.
+            Sorry for the inconvenience and we will work to fix this ASAP.
+          </p>
```

Comment:
Given our target audience is scientists it might make sense to keep them as far removed from GitHub as possible - remember before I knew how to code GitHub / sourceforge were v. scary places. Having concrete actions for them to perform by clicking a button might probably be the best we can do; bar making a "help" website, but I think that's out of scope for now.

Agree with you that we should mention the bug report and improve our copy more. Gonna update; keen to hear what you think.